cmake/Corrosion.cmake

```diff
@@ -105,8 +105,13 @@ function(_add_cargo_build)
         set (build_dir .)
     endif()
 
-    set(link_libs "$<GENEX_EVAL:$<TARGET_PROPERTY:cargo-build_${target_name},CARGO_LINK_LIBRARIES>>")
-    set(search_dirs "$<GENEX_EVAL:$<TARGET_PROPERTY:cargo-build_${target_name},CARGO_LINK_DIRECTORIES>>")
+    # Join the list with `|` as a custom delimiter, since the `;` is expanded when passed
```

Comment:
Did you mean `:` here?
Path can contain `:` on Linux iirc.

Reply:
Yes, I changed to `:`, actually because PATH is `:` delimited. So using the same delimiter for our usecase seems reasonable. I adjusted the wording to make it clearer that file/directory paths cannot contain `:`.